glassfish-tck-runner/pom.template.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <artifactId>weld-core-parent</artifactId>
         <groupId>org.jboss.weld</groupId>
-        <version>5.0.1.Final</version>
+        <version>5.1.2.Final</version>
```

Comment:
I'm not convinced we should be using this version of Weld, we explicitly use 5.0.x in Payara due to breaking changes in Weld

Reply:
yes you are aright, I added the snapshot version for now until I upload to the nexus the final version of our patched weld version